src/NLog.Extensions.Logging/Logging/NLogMessageParameterList.cs

```diff
@@ -93,21 +93,14 @@ private static bool IsValidParameterList(IReadOnlyList<KeyValuePair<string, obje
                 char firstChar = parameterName[0];
                 if (firstChar >= '0' && firstChar <= '9')
                 {
-                    if (!isPositional && i != 0)
-                        isMixedPositional = true;
+                    if (!isPositional)
+                        isMixedPositional = i != 0;
```

Comment:
The semantics changed here. Is that on purpose?

Reply:
Yes the result is the same. But one less condition.